templates/components/itilobject/itilsatisfaction.html.twig

```diff
@@ -46,83 +46,102 @@
         ) }}
 
     {% else %}
+        {% if expired == 1 %}
+            {% set message = __('Satisfaction survey expired, here are the results recorded.') %}
```

Comment:
```suggestion
            {% set message = __('Satisfaction survey expired.') %}
```
I'm not sure about the second part. It probably doesn't need to be explained what is being shown.